apis/swagger.yml

```diff
@@ -1486,31 +1499,6 @@ definitions:
         items:
           $ref: "#/definitions/Node"
 
-  HeartBeatRequest:
```

Comment:
If the HTTP request method could be changed into `POST`, please remain the `HeartBeatRequest ` here. That could be better.

Reply:
done